src/clj/rems/api/applications.clj

```diff
@@ -71,12 +73,49 @@
   (->> (applications/get-all-applications user-id)
        (filter potential-todo?)))
 
+(def ^:private todo-commands
+  #{:application.command/approve
+    :application.command/reject
+    :application.command/revoke ; should not be available in submitted state, but let's keep it here just in case
```

Comment:
The permission to use one of these commands on a submitted application indicates that the application should be shown in the todo list instead of the handled todo list.